sites/next.skeleton.dev/src/content/docs/components/tab/react.mdx

```diff
@@ -0,0 +1,88 @@
+---
+layout: '@layouts/LayoutDoc.astro'
+---
+
+export const components = componentSet;
+
+import { Page as Example } from '@examples/components/tabs/Example.tsx';
+import ExampleRaw from '@examples/components/tabs/Example.tsx?raw';
+import { Page as IconsExample } from '@examples/components/tabs/IconsExample.tsx';
+import IconsExampleRaw from '@examples/components/tabs/IconsExample.tsx?raw';
+import { Page as VerticalIconsExample } from '@examples/components/tabs/VerticalIconsExample.tsx';
+import VerticalIconsExampleRaw from '@examples/components/tabs/VerticalIconsExample.tsx?raw';
+import { Page as StretchExample } from '@examples/components/tabs/StretchExample.tsx';
+import StretchExampleRaw from '@examples/components/tabs/StretchExample.tsx?raw';
+import { Page as JustifyExample } from '@examples/components/tabs/JustifyExample.tsx';
+import JustifyExampleRaw from '@examples/components/tabs/JustifyExample.tsx?raw';
+import { Page as FocusableContentExample } from '@examples/components/tabs/FocusableContentExample.tsx';
+import FocusableContentExampleRaw from '@examples/components/tabs/FocusableContentExample.tsx?raw';
+
+{/* --- */}
+
+<Preview client:load>
+	<Fragment slot="preview">
+		<Example client:load />
+	</Fragment>
+	<Fragment slot="code">
+		<Code code={ExampleRaw} lang="tsx" />
+	</Fragment>
+</Preview>
+
+## Layout
+
+Easily customize tabs content with icons and arrange them in any combination you prefer.
+
+<Preview client:load>
+	<Fragment slot="preview">
+		<IconsExample client:load />
+	</Fragment>
+	<Fragment slot="code">
+		<Code code={IconsExampleRaw} lang="tsx" />
+	</Fragment>
+</Preview>
+
+<Preview client:load>
+	<Fragment slot="preview">
+		<VerticalIconsExample client:load />
+	</Fragment>
+	<Fragment slot="code">
+		<Code code={VerticalIconsExampleRaw} lang="tsx" />
+	</Fragment>
+</Preview>
+
+## Stretch
+
+Tabs can be stretched to fill the available space.
+
+<Preview client:load>
+	<Fragment slot="preview">
+		<StretchExample client:load />
+	</Fragment>
+	<Fragment slot="code">
+		<Code code={StretchExampleRaw} lang="tsx" />
+	</Fragment>
+</Preview>
+
+## Justify
+
+<Preview client:load>
+	<Fragment slot="preview">
+		<JustifyExample client:load />
+	</Fragment>
+	<Fragment slot="code">
+		<Code code={JustifyExampleRaw} lang="tsx" />
+	</Fragment>
+</Preview>
+
+## A11y
+
+The tabs component complies with accessibility guidelines, ensuring an optimal user experience for everyone.
```

Comment:
Suggested Text: "This component adheres to the ARIA APG. Use the preview below to test tabbing and keyboard navigation"